lib/rules/jsx-props-no-spread-multi.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @fileoverview Prevent JSX prop spreading the same expression multiple times
+ * @author Simon Schick
+ */
+
+'use strict';
+
+const docsUrl = require('../util/docsUrl');
+const report = require('../util/report');
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+const messages = {
+  noMultiSpreading: 'Spreading the same expression multiple times is forbidden',
+};
+
+/**
+ * Filter for JSON.stringify that omits circular and position structures.
+ *
+ * @param {string} key
+ * @param {*} value
+ * @returns {*}
+ */
+const propertyFilter = (key, value) => (key !== 'parent' && key !== 'range' && key !== 'loc' ? value : undefined);
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Disallow JSX prop spreading the same expression multiple times',
+      category: 'Best Practices',
+      recommended: true,
+      url: docsUrl('jsx-props-no-spread-multi'),
+    },
+    messages,
+  },
+
+  create(context) {
+    return {
+      JSXOpeningElement(node) {
+        const spreads = node.attributes.filter((attr) => attr.type === 'JSXSpreadAttribute');
+        if (spreads.length < 2) {
```

Comment:
Would be nice to have some stats on how many spreads exists on average per component, though this probably covers ~99.99% of all component instantiations.